packages/react-native-reanimated/jest.config.js

```diff
@@ -5,4 +5,7 @@ module.exports = {
   setupFilesAfterEnv: ['@testing-library/jest-native/extend-expect'],
   testEnvironment: 'node',
   transformIgnorePatterns: [],
+  globals: {
+    ReanimatedError: true
+  }
```

Comment:
Why do we set it to `true` here?

Reply:
It was needed when `ReanimatedError` was declared as a global variable. Now is no longer needed.